src/components/CommentsList.jsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from "react";
+import { getComments } from "../api";
+import CommentCard from "./CommentCard";
+
+export default function CommentsList({ article_id }) {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getComments(article_id)
+      .then((comments) => {
+        console.log(comments, "<----- comments");
+        setComments(comments);
+        setIsLoading(false);
+      })
+      .catch((error) => {
```

Comment:
As mentioned, this catch will never happen even if there is a problem. You'll need to remove the catch from your api file call.